Makefile

```diff
@@ -0,0 +1,64 @@
+LOCAL_IMAGE = acorn_docker:1.0
+REMOTE_IMAGE = merlinran/acorn_docker
```

Comment:
Variables are similar to shell variables, you just need to reference them via `$(var)`, instead of `$${var}`(`${var}` in normal shell, but since we are running shell scripts within `Makefile`, we need to escape `$`, similar to why we have to use `\\` to represent `\` in C string).